traffic_portal/app/src/common/modules/form/deliveryServiceStaticDnsEntry/form.deliveryServiceStaticDnsEntry.tpl.html

```diff
@@ -57,9 +57,12 @@
                 </div>
             </div>
             <div class="form-group" ng-class="{'has-error': hasError(dsStaticDnsEntryForm.address), 'has-feedback': hasError(dsStaticDnsEntryForm.address)}">
-                <label for="address" class="control-label col-md-2 col-sm-2 col-xs-12">Address *</label>
+                <label for="address" class="has-tooltip control-label col-md-2 col-sm-2 col-xs-12">Address *<div class="helptooltip">
+                    <div class="helptext">The Address Rules.<br>1. Type:A_RECORD, Address should be an IPv4 address. <br>2. Type: AAAA_RECORD, Address should be an IPv6 address. <br>3. Type: CNAME, Address must end with a trailing period if type:CNAME. Eg: cdn.test.com. is correct but cdn.test.com will not work</div>
+                </div>
+                </label>
                 <div class="col-md-10 col-sm-10 col-xs-12">
-                    <input id="address" name="address" type="text" class="form-control" ng-model="staticDnsEntry.address" required>
+                    <input id="address" name="address" type="text" class="form-control" ng-model="staticDnsEntry.address" ng-pattern="StaticDNSPattern" required title= "Address must be: an IPv4, if type:A_RECORD; be an IPv6, if type: AAAA_RECORD; end with a trailing period if type:CNAME_RECORD; ">
```

Comment:
`ng-pattern="StaticDNSPattern"`

doesn't that need to come out? ^^

Reply:
I thought I removed it.. 
Updated code.